recipes/arctic/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "arctic" %}
+{% set version = "1.79.2" %}
+{% set sha256 = "9bd9335355d2881c9d52c94cf28faec67a7cf8b547ee6b53834a471b5a4b864d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - decorator
+    - pandas
+    - pymongo >= 3.6.0
```

I've already fixed it in master but this should be:
```suggestion
    - pymongo >=3.6.0
```
(If the formatting is wrong the feedstock creation fails)

---

@xhochy @chrisburr Thanks for your help!

The repo https://github.com/conda-forge/arctic-feedstock is still empty though, is there anything I can do?

---

Not really, I think [this job](https://travis-ci.org/conda-forge/staged-recipes/builds/538290357) should work (there was another version constraint issue and then a network glitch).